client/a8c-for-agencies/sections/marketplace/hosting-overview/hooks/use-hosting-description.ts

```diff
@@ -0,0 +1,39 @@
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+
+/**
+ * Returns hosting name and description with given product slug.
+ * @param slug
+ * @returns
+ */
+export default function useHostingDescription( slug: string ): {
+	name: TranslateResult;
+	description: TranslateResult;
+} {
+	const translate = useTranslate();
+
+	return useMemo( () => {
+		let description = '';
+		let name = '';
+
+		switch ( slug ) {
+			case 'pressable-hosting':
+				name = 'Pressable';
```

Comment:
I am not sure if the Pressable and Wordpress.com needs to be a translatable string.

Reply:
I think it is better to translate.